tests/test_extensions/test_ext_apidoc.py

```diff
@@ -1,23 +1,29 @@
 """Test the sphinx.apidoc module."""
 
 import os.path
-from collections import namedtuple
+from pathlib import Path
+from typing import NamedTuple
 
 import pytest
 
 import sphinx.ext.apidoc
 from sphinx.ext.apidoc import main as apidoc_main
 
 
+class Apidoc(NamedTuple):
+    coderoot: Path
+    outdir: Path
+
+
 @pytest.fixture()
-def apidoc(rootdir, tmp_path, apidoc_params):
+def apidoc(rootdir, tmp_path, apidoc_params) -> Apidoc:
```

Comment:
```suggestion
def apidoc(
    rootdir: Path, 
    tmp_path: Path, 
    apidoc_params: tuple[Sequence[Any], Mapping[str, Any]],
) -> Apidoc:
```

+ add imports + ``from __future__ import annotations``.

While I don't really care about typing inside a test function, I think it's good to type the fixture definitions themselves.